app/models/credential.rb

```diff
@@ -0,0 +1,3 @@
+class Credential < ApplicationRecord
```

Comment:
I'm thinking ahead since modifying database tables later is a little annoying.

Should we lean into the idea of self-serviceability and call this `Configuration`? It could contain credentials, and it could also contain nonsensitive things like:
* Twilio username
* Sync radius
* Reports refresh rate

Reply:
Makes sense. I'll alter this to be more generically usable.